clang/test/CodeGenSYCL/simple-sycl-virtual-function.cpp

```diff
@@ -0,0 +1,68 @@
+// REQUIRES: CPU
+// Adjust when uninitialized global variables are supported on GPU
+
```

Comment:
This test does not reach BE compilers, it is purely to test FE behavior, which should be device-agnostic

```suggestion
```

Reply:
Oh, yes, It's reduced E2E test so I forgot to remove these lines